From fd09cbe76763801691c168bdf51bfcf3581a861c Mon Sep 17 00:00:00 2001
From: Simon Horman <horms+renesas@verge.net.au>
Date: Wed, 3 Jul 2013 13:38:21 +0900
Subject: [PATCH 0944/1083] Revert "drm/rcar-du: Use the mode active size in
 mode set"

This reverts commit 8f859ab75084dc649f9a0c2ad6f113e1600f1e1a.
(cherry picked from commit 25447c2ef883d4075956fd335ac4fd42ef571a0f)

Signed-off-by: Ryo Kataoka <ryo.kataoka.wt@renesas.com>
---
 drivers/gpu/drm/rcar-du/rcar_du_crtc.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/rcar-du/rcar_du_crtc.c b/drivers/gpu/drm/rcar-du/rcar_du_crtc.c
index 08a69db..1b49b54 100644
--- a/drivers/gpu/drm/rcar-du/rcar_du_crtc.c
+++ b/drivers/gpu/drm/rcar-du/rcar_du_crtc.c
@@ -412,8 +412,8 @@ static int rcar_du_crtc_mode_set(struct drm_crtc *crtc,
 
 	rcrtc->plane->src_x = x;
 	rcrtc->plane->src_y = y;
-	rcrtc->plane->width = mode->hdisplay;
-	rcrtc->plane->height = mode->vdisplay;
+	rcrtc->plane->width = crtc->fb->width;
+	rcrtc->plane->height = crtc->fb->height;
 
 	rcar_du_plane_compute_base(rcrtc->plane, crtc->fb);
 
-- 
1.8.3.2

